test/services/reports_tar_reader_test.rb

```diff
@@ -11,7 +11,7 @@ class ReportsTarReaderTest < ActiveSupport::TestCase
         XccdfReportParser.new(
           report,
           'account' => accounts(:test).account_number,
-          'id' => @host_id,
+          'id' => ::UUID.generate,
```

Comment:
I don't understand why this change is necessary - I assume some test failed, but I don't understand why.

Reply:
`@host_id` was nil here - so since `message['id']`was nil, it failed